src/static/riot/analytics/analytics.tag

```diff
@@ -0,0 +1,540 @@
+            </div>
+        </div>
+        <div class="date-input">
+            <h3>End Date</h3>
+            <div class="ui calendar" ref="end_calendar">
+                <div class="ui input left icon">
+                    <i class="calendar icon"></i>
+                    <input type="text" placeholder="{ end_date}">
+                </div>
+            </div>
+        </div>
+    </div>
+
+    </div>
+
+    <h3>Chart Resolution</h3>
+    <div class="ui blue buttons">
+        <button class="ui button" onclick="{ update_chart_resolution }" id="day-units">Daily</button>
+        <button class="ui button" onclick="{ update_chart_resolution }" id="week-units">Weekly</button>
+        <button class="ui button" onclick="{ update_chart_resolution }" id="month-units">Monthly</button>
+    </div>
+
+
+    <div class="ui top attached tabular menu">
+        <a class="active item" data-tab="competitions">Competitions</a>
+        <a class="item" data-tab="submissions">Submissions</a>
+        <a class="item" data-tab="users">Users</a>
+    </div>
+
+        <div class="ui bottom attached active tab segment" data-tab="competitions">
+            <div class="ui statistics">
+                <div class="statistic">
+                    <div class="value">
+                        {competitions}
+                    </div>
+                    <div class="label">
+                        Competitions Created
+                    </div>
+                </div>
+                <div class="statistic">
+                    <div class="value">
+                        {competitions_published}
+                    </div>
+                    <div class="label">
+                        Competitions Published
+                    </div>
+                </div>
+            </div>
+            <div class='chart-container'>
+                <canvas ref="competition_chart"></canvas>
+            </div>
+        </div>
+
+        <div class="ui bottom attached tab segment" data-tab="submissions">
+            <div class="ui statistic">
+                <div class="value">
+                    {submissions_made}
+                </div>
+                <div class="label">
+                    Submissions Made
+                </div>
+            </div>
+            <div class='chart-container'>
+                <canvas ref="submission_chart"></canvas>
+            </div>
+        </div>
+
+        <div class="ui bottom attached tab segment" data-tab="users">
+            <div class="ui statistic">
+                <div class="value">
+                    {users_total}
+                </div>
+                <div class="label">
+                    Users Joined
+                </div>
+            </div>
+            <div class='chart-container'>
+                <canvas ref="user_chart"></canvas>
+            </div>
+        </div>
+
+        <button class="ui big blue button" onclick="{ download_analytics_data }">Download</button>
+    <script>
```

Comment:
Should try and keep a consistent variable instantiation convention through the whole tag, either `let`/`const` or `var`

Reply:
Noted. Changed to let where appropriate.